temporalio/converter.py

```diff
@@ -23,6 +23,7 @@
     ClassVar,
     Dict,
     List,
+    Literal,
```

Comment:
Confirmed that `typing.Literal` is present in 3.8+ so it is safe for use in all versions we currently support (it wasn't present in the oldest supported Python version when we first developed this)